packages/assets-controllers/src/NftController.test.ts

```diff
@@ -1955,6 +1964,147 @@ describe('NftController', () => {
         tokenURI: '',
       });
     });
+
+    it('should, when passed a networkClientId, add an NFT with the correct chainId and metadata fetched using networkClientId', async () => {
```

Comment:
```suggestion
    it('should add an NFT with the correct chainId and metadata when passed a networkClientId, ', async () => {
```

not important. just feels wordy

Reply:
done here: https://github.com/MetaMask/core/pull/1698/commits/5fd50ec7deb2b8e06f3221c78abd7363df0d7039